### PR TITLE
#14-1 admin-function LAST-PR-#9

### PR DESCRIPTION
**Pull request by kokioba95**
## Description / Purpose (Checkpoint)
This PR confirmation is the administrator function (& Associated page layout).

I fixed the problem.
< https://docs.google.com/spreadsheets/d/1Ear__g58irVlaI_tpzWUkf__JRcBz8u2d2DdRYKMiDk/edit?usp=sharing > <Backlog # 11>

# # To test this PR
【Admin】
1. Go to the login page (example: http: // localhost: 8086 / login).
2. Input user credentials (Admin)
3. Click Login
4. my_profile page
5. Go to the top page
6. Go to the event list page
7. Go to event info page
8. Press the Delete button.
9. (Go to Event list)
【User】
1. Go to the login page (example: http: // localhost: 8086 / login).
2. Input user credentials (User)
3. Click Login
4. my_profile page
5. Go to the top page
6. Go to the event list page
7. Go to event info page
8. Confirm that there is no Delete button.

**command**
# Php artisan migrate: reset

**screenshot**
 -< https://drive.google.com/drive/folders/1UX_at7VJu0OeH5Mb7Eo7ULK3hSw7Ws-W?usp=sharing >

### CODE REVIEW

**Review by hayato0309 — APPROVED** [review]

File: app/Http/Controllers/AdminController.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    }
+    public function admin_login()
+    {
+        return view('admin/admin_login');
+    }
+    protected function guard()
+    {
+        return \Auth::guard('admin'); //管理者認証のguardを指定
+    }
+    //public function logout(Request $request)
```
Comment (AubreyV) [inline app/Http/Controllers/AdminController.php]: Please do not make it a habit to push commented codes. You can push comments that explains a code but do not push commented codes.

File: app/Http/Controllers/Auth/RegisterController.php [hunk 2]
```diff
@@ -71,6 +72,7 @@ protected function create(array $data)
             'group_name' => $data['group_name'],
             'email' => $data['email'],
             'password' => Hash::make($data['password']),
+            // 'is_admin' => $data['is_admin']->nullable,
```
Comment (AubreyV) [inline app/Http/Controllers/Auth/RegisterController.php]: Please do not make it a habit to push commented codes. You can push comments that explains a code but do not push commented codes.

File: resources/views/admin/admin_my_profile.blade.php [hunk 3]
```diff
@@ -0,0 +1 @@
+fghjkl
```
Comment (AubreyV) [inline resources/views/admin/admin_my_profile.blade.php]: What is this? and why is this here?

File: resources/views/event_info.blade.php [hunk 4]
```diff
@@ -6,13 +6,25 @@
         <div class="col-sm-1">
         </div>
         <div class="col-sm-10" style="background-color:white">
+        <div>
```
Comment (AubreyV) [inline resources/views/event_info.blade.php]: Whats inside this div?

File: resources/views/event_info.blade.php [hunk 5]
```diff
@@ -6,13 +6,25 @@
...
                 <div class="col-sm-10">
-                    <div class="col-sm-12">
+                <div class="row">
+                    <div class="col-sm-8">
                         <h1>{{ $event->event_title }}</h1>
                     </div>
+                    <div class="col-sm-4">
+                        @if (auth()->user()->is_admin)
+                            <a class="btn btn-primary" href="/{{$event->id}}/delete" role="button">Delete</a>
+                        @else
```
Comment (AubreyV) [inline resources/views/event_info.blade.php]: What should be inside this @else?

File: resources/views/event_info.blade.php (hunk 4 above)
Comment (kokioba95) [reply resources/views/event_info.blade.php]: I made a correction.

File: app/Http/Controllers/Auth/RegisterController.php (hunk 2 above)
Comment (kokioba95) [reply app/Http/Controllers/Auth/RegisterController.php]: I understood.
I deleted the code.

File: app/Http/Controllers/AdminController.php (hunk 1 above)
Comment (kokioba95) [reply app/Http/Controllers/AdminController.php]: All files were deleted because they were unnecessary.
I will try to confirm the deletion in the future.

File: resources/views/admin/admin_my_profile.blade.php (hunk 3 above)
Comment (kokioba95) [reply resources/views/admin/admin_my_profile.blade.php]: All files were deleted because they were unnecessary.
I will try to confirm the deletion in the future.

**Review by AubreyV — APPROVED** [review]
